lib/bots/versionbot.ts

```diff
@@ -26,10 +26,9 @@ import { cleanup, track } from 'temp';
 import * as GithubApiTypes from '../apis/githubapi-types';
 import { ProcBot } from '../framework/procbot';
 import { ProcBotConfiguration } from '../framework/procbot-types';
-import { FlowdockEmitRequestContext } from '../services/flowdock-types';
-import { GithubCookedData, GithubEmitRequestContext, GithubHandle,
-    GithubRegistration } from '../services/github-types';
-import { ServiceEmitRequest, ServiceEmitResponse, ServiceEvent } from '../services/service-types';
+import { GithubError } from '../services/github';
```

Comment:
Yes, it's a class implementation, which can't exist in ambient declarations. It therefore has to be defined in a TS source file and not definition file.